presentation/src/main/java/com/goms/presentation/view/outing/OutingFragment.kt

```diff
@@ -33,14 +33,19 @@ class OutingFragment : Fragment() {
     private val outingViewModel by viewModels<OutingViewModel>()
     private lateinit var binding: FragmentOutingBinding
 
+
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?)
```

Comment:
끝나는 괄호를 저렇게 하니까 코드가 안이쁩니다.
```suggestion
        savedInstanceState: Bundle?
) : View {
```

Reply:
넵 수정하겠습니다